components/logins/android/src/main/java/mozilla/appservices/logins/DatabaseLoginsStorage.kt

```diff
@@ -29,6 +29,17 @@ class DatabaseLoginsStorage(private val dbPath: String) : AutoCloseable, LoginsS
         return handle
     }
 
+    /**
+     * Return the raw handle used to reference this logins database.
+     *
+     * Generally should only be used to pass the handle into `SyncManager.setLogins`.
+     *
+     * Note: handles do not remain valid after locking / unlocking the logins database.
```

Comment:
Does that mean Lockwise will need to call `setLogins` again after unlocking? It sounds like yes, from this comment and the one above `ClosedEngine`, but just double-checking.

Reply:
Yes. The logins api's lock/unlock system is the reason for this being so convoluted, really, although it's nice to give the other engines the flexibility to, well, close ever.